### ISSUE TITLE
10.11.X support

### ISSUE THREAD

**Issue by huangwei0418** [vit9696/AppleALC#41]
Apple has changed the AppleHDA binary a lot, so if you are trying to use bcc’s patch-hda or toleda’s audioALC scripts, you will encounter with some issues.And of course you can manually modify these script to make them work fine with OS X Capitan

Apple added a new codec id 0x11D41983 and removed 0x10EC0884, 0x1AEC87FF (or more).

 I have a legacy version of ALC888 (rev0x0202) which is not supported by default, I used legacy patch rev0x0201 patch from toleda.
For the AppleHDA binary, I replaced 0x11D41984 with 0x10EC0888.
Besides you have to disable the new codec 0x11D41983, otherwise AppleHDA.kext will be loaded without any sound.

To sum up, if you are using 0x11D41984 codec patch and suffering no sound, please try to disable this new codec id 0x11D41983

kext patch:
Name:
AppleHDA
Find
8319D411
Replace
00000000

**Comment by vandroiy2013** [comment]
ALC888 is masked under ALC885. In this case we don't need 11D41983 zeroing. Give me please info.plist and terminal output made by attached script. I will add support for your revision 0x100202.
[Codec-Info.command.zip](https://github.com/vit9696/AppleALC/files/243566/Codec-Info.command.zip)
